service/src/main/java/bio/terra/pipelines/service/ImputationService.java

```diff
@@ -85,6 +96,41 @@ public UUID createImputationJob(
     return jobBuilder.submit();
   }
 
+  /**
+   * Temporary placeholder for constructing custom imputation inputs - for now just logs the input
+   * keys and values. In the future (TSPS-169), this will construct the object that is included in
+   * the call to CBAS.
+   *
+   * @param imputationPipelineName - the name of the pipeline
+   * @param userProvidedPipelineInputs - the user-provided inputs
+   * @return Map<String, Object> allPipelineInputs - the combined inputs
+   */
+  public Map<String, Object> constructImputationInputs(
```

Comment:
currently there's not really imputation-specific stuff in here but I think when the CBAS call formatting is done it will. but that might happen in a different method?

Reply:
yeah, at this point that makes sense. I'll refactor